inst/www/shared/datepicker/css/bootstrap-datepicker3.css

```diff
@@ -103,7 +103,7 @@
 .datepicker table tr td.disabled,
 .datepicker table tr td.disabled:hover {
   background: none;
-  color: #777777;
+  color: #AAA;
```

Comment:
I think it would be better to do this in shiny.css -- in general we try not to modify third-party library code, to make it easier to upgrade in the future.